packages/services/f-braze-adapter/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@justeat/f-braze-adapter",
   "description": "Fozzie Metadata Service",
-  "version": "4.2.0",
+  "version": "4.3.0",
```

Comment:
IMO, this should be a major version bump as it is a backwards compatibility break.